src/scss/_variables.scss

```diff
@@ -1,4 +1,4 @@
-$lightgray: rgba(245, 245, 245, 0.25);
+$lightgray: #3d3d3d;
```

Comment:
This probably isn't intentional, but 3d3d3d isn't a light gray and that "broke" the collapse boxes and the text when the bucket is collapsed...